doc/classes/ProjectSettings.xml

```diff
@@ -2928,7 +2927,7 @@
 		</member>
 		<member name="xr/openxr/foveation_level" type="int" setter="" getter="" default="&quot;0&quot;">
 			Applied foveation level if supported: 0 = off, 1 = low, 2 = medium, 3 = high.
-			[b]Note:[/b] Only works on compatibility renderer.
+			[b]Note:[/b] Only works on compatibility renderer. On platforms other than Android, if [member rendering/anti_aliasing/quality/msaa_3d] is enabled, this feature will be disabled. For an alternative, set [member rendering/vrs/mode] to [constant Viewport.VRS_XR]
```

Comment:
Oopsie.
```suggestion
			[b]Note:[/b] Only works on the Compatibility rendering method. On platforms other than Android, if [member rendering/anti_aliasing/quality/msaa_3d] is enabled, this feature will be disabled. For an alternative, set [member rendering/vrs/mode] to [constant Viewport.VRS_XR].
```
As for the "Compatibility renderer" change, see https://github.com/godotengine/godot/pull/93673#discussion_r1657940501

Reply:
Thank you! I agree. I also changed the "compatibility renderer" of the previous setting, since they are part of the same system (foveated rendering)